CONTRIBUTING.md

```diff
@@ -3,6 +3,64 @@
 This documention is a work-in-progress that we'll be filling in as our best
 practices solidify.
 
+## Setting local environment variables
+
+1. Copy the sample env file
+
+    `cp .env.sample .env.local`
+
+1. Modify variables in .env.local, e.g. uncommenting the `EXPERIMENTER_API_CALL`
+   line will switch from the default of live experiments to preview experiments
+
+## Running the development server
+
+```bash
+npm run dev
+```
+
+## Enabling Auth0 locally
+
+Local development currently has Auth0 turned off.
+It’s possible to enable Auth0 locally (if desired) for testing, by setting the 
+correct Auth0 values in `.env.local`, and using
+
+```bash
+npm run start
+```
+to run the build, which simulates the prod environment.
+
+To test Auth0 locally, please create an account at https://auth0.com/, create a new app,
+and add the correct properties to `.env.local`:
+  
+Auth0 properties
+# A long, secret value used to encrypt the session cookie 
+# (use [openssl rand -hex 32] to generate a 32 bytes value)
+AUTH0_SECRET='LONG_RANDOM_VALUE'
+
+# The base url of the application
+AUTH0_BASE_URL='http://localhost:3000'
+
+# The url of the Auth0 tenant domain
+AUTH0_ISSUER_BASE_URL='AUTH0_TENANT_URL'
+
+# The application's Client ID
+AUTH0_CLIENT_ID='YOUR_AUTH0_CLIENT_ID'
+
+# The application's Client Secret
+AUTH0_CLIENT_SECRET='YOUR_AUTH0_CLIENT_SECRET'
+
+Within the Auth0 dashboard, set the following properties:
+
+```
+Application Login URI: https://fxms-skylight.netlify.app/api/auth/login
+
+Allowed Callback URLs: http://localhost:3000/api/auth/callback
+
+Allowed Logout URLs: https://fxms-skylight.netlify.app/, http://localhost:3000
+```
```

Comment:
It's not obvious to me why any of the local development callback URLs would points to Netlify. If this is truly necessary, can you clarify why in the text?

Reply:
So I just tested this && the only one that's actually required to point to `https://fxms-skylight.netlify.app/` is the login URI; Auth0 won't accept a non-https URL or a localhost for that. I can put that explanation in the doc :)